### ISSUE TITLE
Docker meetup feedback - lesson #1

### ISSUE THREAD

**Issue by subfuzion** [subfuzion/docker-voting-app-nodejs#6]
Please leave feedback here for the [Docker Mountain View meetup](https://www.meetup.com/Docker-Mountain-View/events/245861049/) on January 17, 2018. Feel free to note the things that were especially helpful and things that could use more explanation. Suggestions are welcome.

UC Davis has offered to provide a promo code to take the course for free once it is available on Coursera (April-May timeframe) for your participation here in providing feedback and offering to participate in a forum (here in GitHub issues or maybe a live discussion channel) to help other new learners when they start taking the course.

**Comment by sthyagarajan** [comment]
@subfuzion Lot of great things on Prometheus, would have loved to have a introduction on Docker before the monitoring session for those who are new.

**Comment by anandraghunathan** [comment]
@subfuzion Came here to get an introduction to Docker, my bad that I missed the earlier meetup. The session was useful, perhaps advanced for a newbie :)

**Comment by vnosovsky** [comment]
can you share slides please?

**Comment by mghatak** [comment]
@subfuzion Great introduction! Hope to see more like this.

**Comment by iwilsonq** [comment]
fabulous talk, how can i learn about future sessions?

**Comment by subfuzion** [comment]
Here is the link to the slides: https://docs.google.com/presentation/d/1UD3xeLE04TkU65kBy5thnTEls6tOAXRq24JL2G8nMXk/edit?usp=sharing

**Comment by subfuzion** [comment]
>> would have loved to have a introduction on Docker before the monitoring session for those who are new

@sthyagarajan that's good feedback. Originally the monitoring piece of this was supposed to come later, so I will revisit the order for the next few sessions.

**Comment by subfuzion** [comment]
> fabulous talk, how can i learn about future sessions?

@iwilsonq - thank you! Please stay tuned to the meetup schedule [here](https://www.meetup.com/Docker-Mountain-View/). We're having a Docker beer night next week and the next lesson in this series is next month.

**Comment by subfuzion** [comment]
> Great introduction! Hope to see more like this.

@mghatak Thank you! Yes, the series will continue over upcoming meetups, which will be posted [here](https://www.meetup.com/Docker-Mountain-View/).

**Comment by subfuzion** [comment]
> The session was useful, perhaps advanced for a newbie :)

@anandraghunathan Yes, I will reconsider the ordering of the material, slow down the pace a bit, and provide links to more amplifying documentation.

**Comment by sahriyarm** [comment]
Hi Tony, it was pleasure to meet you 2 days in a row, will definitely be early adaptor of course! Thanks!

**Comment by subfuzion** [comment]
@sahriyarm Awesome! And likewise, it's a pleasure. I enjoyed the meetup on istio this evening. See you next Thursday if you join us at Stein's for the [Docker beer social](https://www.meetup.com/Docker-Mountain-View/events/245861186/)!

**Comment by sahriyarm** [comment]
Definitely @subfuzion! Thanks for pointing! See you!

**Comment by pkoppa** [comment]
Learning Docker directly a Docker captain is really really exciting. Looking forward for future sessions.
First session was amazing.

I would like to see a small assignment at the end of session to take back home which will help in exploring more of covered topic.
